packages/aws-cdk/lib/api/toolkit-info.ts

```diff
@@ -114,3 +193,18 @@ function getOutputValue(stack: aws.CloudFormation.Stack, output: string): string
   }
   return result;
 }
+
+const DEFAULT_REPO_LIFECYCLE = {
+  rules: [
+    {
+      rulePriority: 100,
+      description: 'Retain only 5 images',
+      selection: {
+        tagStatus: 'any',
+        countType: 'imageCountMoreThan',
+        countNumber: 5,
```

Comment:
That seems excessively low to me. Especially since it is not user-customizable.

Reply:
Yes, it should be like 3 months or something

Reply:
In what scenario will those old images be used though? Every repository is tied to a single stack in a single region, and once the stack is updated all references will point to the new image, which immediately follows deployment.

I can see retaining two images for quick rollback purposes, and I put a little more margin on there "just because" that feels safe.

But 3 months? Don't forget that these images cost money as well. We already have a user starting to complain about the size of their asset bucket.

Reply:
Keep in mind, we can't tell if these images are currently deployed somewhere.  So if 5 builds happen that fail testing and never get deployed, my production image will be deleted.  Time-based policy means I'm banking on deploying at least one a quarter.  I would actually say 6 months is safer...